app/controllers/response_controller.rb

```diff
@@ -181,6 +181,13 @@ def create
     msg = "Your response was successfully saved."
     error_msg = ""
     @response.email
+    begin
+      Response.compare_scores(@response, @questionnaire)
+
+      msg = "Your response was successfully saved."
```

Comment:
At this point, the response record in db has already been created. So there's no point to rescue and give the error message saying "An error occurred..."’